src/commands/cli/release/build.ts

```diff
@@ -0,0 +1,239 @@
+import { arrayWithDeprecation, Flags, SfCommand, Ux } from '@salesforce/sf-plugins-core';
+import { exec, ExecOptions, set } from 'shelljs';
+import { ensureString } from '@salesforce/ts-types';
+import { Env } from '@salesforce/kit';
+import { Octokit } from '@octokit/core';
+import { bold } from 'chalk';
+import { Messages, SfError } from '@salesforce/core';
+import { PackageRepo } from '../../../repository';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'cli.release.build');
+
+export default class build extends SfCommand<void> {
+  public static readonly description = messages.getMessage('description');
+  public static readonly summary = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly aliases = ['cli:latestrc:build'];
+  public static readonly flags = {
+    'start-from-npm-dist-tag': Flags.string({
+      summary: messages.getMessage('flags.startFromNpmDistTag'),
+      // default: 'latest-rc', // TODO: Will need to update this in GHA before next RC. `exactlyOne` does not work wellwith defaults
+      char: 'd',
+      aliases: ['rctag'],
+      deprecateAliases: true,
+      exactlyOne: ['start-from-npm-dist-tag', 'start-from-github-ref'],
+    }),
+    'start-from-github-ref': Flags.string({
+      summary: messages.getMessage('flags.startFromGithubRef'),
+      char: 'g',
+      exactlyOne: ['start-from-npm-dist-tag', 'start-from-github-ref'],
+    }),
+    'build-only': Flags.boolean({
+      summary: messages.getMessage('flags.buildOnly'),
+      default: false,
+    }),
+    resolutions: Flags.boolean({
+      summary: messages.getMessage('flags.resolutions'),
+      default: true,
+      allowNo: true,
+    }),
+    only: arrayWithDeprecation({
+      summary: messages.getMessage('flags.only'),
+    }),
+    'pinned-deps': Flags.boolean({
+      summary: messages.getMessage('flags.pinnedDeps'),
+      default: true,
+      allowNo: true,
+    }),
+    patch: Flags.boolean({
+      summary: messages.getMessage('flags.patch'),
+      exclusive: ['prerelease'],
+    }),
+    prerelease: Flags.string({
+      summary: messages.getMessage('flags.prerelease'),
+      exclusive: ['patch'],
+    }),
+    snapshot: Flags.boolean({
+      summary: messages.getMessage('flags.snapshot'),
+    }),
+    schema: Flags.boolean({
+      summary: messages.getMessage('flags.schema'),
+    }),
+  };
+
+  public async run(): Promise<void> {
+    // I could not get the shelljs.exec config { fatal: true } to actually throw an error, but this works :shrug:
+    set('-e');
+
+    let auth: string;
+    const { flags } = await this.parse(build);
+
+    if (flags.prerelease === 'true' || flags.prerelease === 'false') {
+      throw new SfError(
+        'The prerelease flag is not a boolean. It should be the name of the prerelease tag, examples: dev, alpha, beta'
+      );
+    }
```

Comment:
I don't think this is necessary.  We don't do boolean like that anyway.

Reply:
Yea, that is fair. I just thought this could prevent an odd edge case. But it is unlikely that we will be calling this from our machines anyway. Most likely it will be from a `workflow_dispatch` that can help prevent this. Will delete.